module/Application/src/Application/View/Helper/Acl.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+namespace Application\View\Helper;
+
+use Zend\View\Helper\AbstractHelper;
+
+
+class Acl extends AbstractHelper
+{
+
+    /**
+     * Service locator
+     *
+     * @var \Zend\ServiceManager\ServiceLocatorInterface
+     */
+    protected $locator;
+
+    /**
+     * Acl
+     */
+    protected $acl;
+
+    public function isAllowed($resource, $operation)
+    {
+        return $this->acl->isAllowed(
+            $this->getRole(),
+            $resource,
+            $operation
+        );
+    }
+
+    /**
+     * Get the current user's role.
+     *
+     * @return User|string
+     */
+    public function getRole()
+    {
+        return $this->getServiceLocator()->get('user_role');
+    }
+
+    /**
+     * Returns the Acl for a specific module
+     *
+     * @param string $factory Acl factory to load
+     *
+     * @return \Application\View\Helper\Acl
+     */
+    public function __invoke($factory)
+    {
+        $this->acl = $this->getServiceLocator()->get($factory);
```

Comment:
It might be a good idea to strictly check if this is indeed an ACL instance, and throw an `InvalidArgumentException` if it is not.